Makefile.in

```diff
@@ -349,7 +349,7 @@ libaddon: exports
 	$(MAKE) -C lib/addons/library.kodi.inputstream
 addon-dev-kit:
 	mkdir -p xbmc/addons/kodi-addon-dev-kit/build
-	cd xbmc/addons/kodi-addon-dev-kit/build; $(CMAKE) -DCMAKE_INSTALL_PREFIX=$(DESTDIR)$(prefix) ..
+	cd xbmc/addons/kodi-addon-dev-kit/build; cmake -DCMAKE_INSTALL_PREFIX=$(DESTDIR)$(prefix) ..
```

Comment:
this doesn't look correct. Or must addon-dev-kit be built for native platform instead of target?
When cross compiling, this will build for native/host platform, which seems wrong to me.